### ISSUE TITLE
Dead project? I hope not

### ISSUE THREAD

**Issue by TheNomad11** [corrad1nho/qomui#140]
While Windows and Mac users have lots of options, there are no decent openvpn apps for Linux users. One of the main issues are the leaks. I was happy when I found Qomui, so I hope this project is still being maintained. Last updates are from more than a half year ago, many issues are not getting answers...

**Comment by Ph0rk0z** [comment]
Yes and I can't use the built in firewall/dnsmasq because I use my own FW and dnscrypt. So blocking when not connected isn't working but (re) connecting is 10x more reliable than using built in VPN functions.

Of all things, how can linux have worse vpn support than windows? I'm saddened that the dev has abandoned it. We need a fork just like opensnitch.

**Comment by stdcerr** [comment]
see WireGuard as a great VPN alternative for Linux: https://wiki.archlinux.org/index.php/WireGuard

**Comment by TheNomad11** [comment]
> see WireGuard as a great VPN alternative for Linux: https://wiki.archlinux.org/index.php/WireGuard

The problem is that there is no Wireguard client for Linux with GUI, only nerdy command line with no auto reconnect etc, it seems

**Comment by Ph0rk0z** [comment]
This program could easily support wireguard. https://client.pritunl.com/ is the closest runner up, don't know how paid vpn apps are because I don't buy their vpns.

Command line utilities and scripts work great for a static environment, not so much for vpn hopping

**Comment by TheNomad11** [comment]
I tried it many times but was never happy with it. It suffers from ip-leaking, especially ip v6
